python/nvtext.py

```diff
@@ -233,6 +233,31 @@ def replace_tokens(strs, tgts, repls, delimiter=None):
     return rtn
 
 
+def normalize_spaces(strs):
+    """
+    Remove extra whitespace between tokens and trim whitespace
+    from the beginning and the end of each string.
+
+    Parameters
+    ----------
+    strs : nvstrings
+        The strings for this operation.
+
+    Examples
+    --------
+    >>> import nvstrings, nvtext
+    >>> s = nvstrings.to_device(["hello \t world"," test string  "])
+    >>> n = nvtext.normalize_spaces(s)
+    >>> print(n)
+    ["hello world", "test string"]
+
+    """
+    rtn = pyniNVText.n_normalize_spaces(strs)
```

Comment:
Would it be difficult to provide in your arguments what you consider a space ?

There may be workflows where you may not want to get rid of `\t` or `\n` as examples.  Everything else looks good to me. 

I get that its similar to , https://github.com/rapidsai/custrings/issues/368 but still wondering why we will have to implementations ?

Reply:
I'm trying to match existing normalize-space functions like the following: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#normalizeSpace-java.lang.String-
I think it would be confusing behavior to provide the space character. I would recommend this be opened as a separate feature request if this really is necessary.

Not sure you have the right issue reference since 368 refers to nvcategory keys documentation. What two implementations are you talking about?

Reply:
I meant #374 , Edited my comment. Okay, mapping the API makes sense, approving it.